rust/query/typeql_delete.rs

```diff
@@ -22,85 +22,44 @@
 
 use crate::common::token::Command::Delete;
 use crate::{
-    write_joined, ErrorMessage, Insertable, Query, ThingVariable, TypeQLMatch, TypeQLUpdate,
-    UpdateQueryBuilder,
+    write_joined, ErrorMessage, Query, ThingVariable, TypeQLMatch, TypeQLUpdate,
+    UpdateQueryBuilder, Writable,
 };
 use std::fmt;
 
 #[derive(Debug, Eq, PartialEq)]
 pub struct TypeQLDelete {
-    pub match_query: Option<TypeQLMatch>,
+    pub match_query: TypeQLMatch,
     pub variables: Vec<ThingVariable>,
 }
```

Comment:
Not actually optional! I copypasted the structure from Insert and did not notice the difference. Same in Update.